pkg/providers/storage.go

```diff
@@ -5,14 +5,36 @@ import (
 	"github.com/weaveworks/ignite/pkg/constants"
 	"github.com/weaveworks/ignite/pkg/storage"
 	"github.com/weaveworks/ignite/pkg/storage/cache"
+	"github.com/weaveworks/ignite/pkg/storage/manifest/raw"
+	"github.com/weaveworks/ignite/pkg/storage/sync"
+	"github.com/weaveworks/ignite/pkg/storage/watch"
 )
 
 // Storage is the default storage implementation
 var Storage storage.Storage
+var SS *sync.SyncStorage
 
 func SetCachedStorage() error {
 	Storage = cache.NewCache(
 		storage.NewGenericStorage(
 			storage.NewDefaultRawStorage(constants.DATA_DIR), scheme.Serializer))
 	return nil
 }
+
+// TODO: Special constructor for this setup
+func SetTestManifestStorage() error {
```

Comment:
remove `Test` from here, and call this when running `ignite daemon`

Reply:
Done